docs/source/usage_guides/accelerate_training.mdx

```diff
@@ -155,6 +155,26 @@ The default value of *epsilon* is `1e-6` for the Habana fused ADAM optimizer, wh
 </Tip>
 
 
+## Improving Training Performance with HPU Graphs
```

Comment:
I would move this section above `## Custom Operators` at line 140 because here it's in the middle of this one.